### ISSUE TITLE
save cedict_ts.u8 to .obsidian

### ISSUE THREAD

**Issue by chu-shen** [tadashi-aikawa/obsidian-various-complements-plugin#169]
Hide this file to avoid accidental deletion👍
`./cedict_ts.u8` -> `./.obsidian/cedict_ts.u8`

**Comment by suliveevil** [comment]
Good idea！

**Comment by tadashi-aikawa** [comment]
I have a feeling there was a reason I could not do so, but I will check again.

**Comment by suliveevil** [comment]
How about this idea:

manage completion sources
```text
.obsidian/various-complements/dict/
.obsidian/various-complements/snippets/
.obsidian/various-complements/thesaurus/
……
```

**Comment by tadashi-aikawa** [comment]
> How about this idea:

We can't edit the files under the `.obsidian` directory on mobile. However, it doesn't need to edit `cedict_ts.u8`, so there is the possibility for `cedict_ts.u8`.

**Comment by suliveevil** [comment]
R/W rights is annoying on iOS/iPadOS, so maybe use plugin's own folder is better:

```text
.ovc/dict/
.ovc/snippets/
.ovc/thesaurus/
……
```

**Comment by tadashi-aikawa** [comment]
Please set your favorites paths.

![image](https://user-images.githubusercontent.com/9500018/192921964-4150edea-968d-4720-9cfc-7aa287bcba98.png)

**Comment by tadashi-aikawa** [comment]
@chu-shen @suliveevil 
Released in [v8.0.0-beta1](https://github.com/tadashi-aikawa/obsidian-various-complements-plugin/discussions/183) 🚀 

If you use BRAT, please confirm it. If not, please wait to release it as official, v8.0.0.

**Comment by tadashi-aikawa** [comment]
@chu-shen @suliveevil 
Released in [v8.0.0](https://github.com/tadashi-aikawa/obsidian-various-complements-plugin/releases/tag/8.0.0) 🚀
